### ISSUE TITLE
mkdir/rmdir: Tab-completion only works on the first argument, but not on the second argument.

### ISSUE THREAD

**Issue by gaeulbyul** [chrisant996/clink#252]
When I try to create multiple directories with the "mkdir" command or remove it with the "rmdir" command, clink provides tab-completion on a first argument but not on later arguments.

```bat
:: Say we already have "foo\" directory
:: Both commands are valid but clink only provides tab-completion (foo\ in here) on the first argument,
:: but not on the second argument.
MKDIR foo\bar foo\baz
MKDIR foo\bar, foo\baz

:: rmdir also have same problem. Both are valid but clink doesn't provide tab-completion on the second argument.
RMDIR foo\bar foo\baz
RMDIR foo\bar, foo\baz
```

**Comment by chrisant996** [comment]
Interesting, thanks for noticing that.

The commands are documented as accepting only one argument, and so Clink only performed completion for one argument.

But the commands actually accept multiple arguments.  I've changed the argmatchers for the commands to perform completion on multiple arguments.

**Comment by chrisant996** [comment]
Fixed in 0ecb57dd0bb7d15f73d4ee59cf5cf6b3ba1bbf48.
